Core/include/Acts/Seeding/SeedFinderUtils.hpp

```diff
@@ -65,6 +66,22 @@ void transformCoordinates(std::vector<external_spacepoint_t*>& vec,
                           external_spacepoint_t& spM, bool bottom,
                           std::vector<LinCircle>& linCircleVec,
                           callable_t&& extractFunction);
+
+/// @brief Check the compatibility of spacepoint coordinates in xyz assuming the Bottom-Middle direction with the strip meassument details
+///
+/// @tparam external_spacepoint_t The external spacepoint type.
+/// @tparam sp_range_t container type for the space point collections.
+///
+/// @param[in] m_config seedfinder config containing the delegate to the strip measurement details
+/// @param[in] spacepointPosition Spacepoint coordinates in xyz plane.
```

Comment:
Sorry, I forgot to add the description of sp parameter:
 `/// @param[in] sp Input spacepoint that will be used in the check.`